docs/docs/system-administration/saml.md

```diff
@@ -89,5 +89,5 @@ To add support for other canonicalization methods, set the `EXTRA_ALLOWED_CANONI
 Example:
 
 ```
-EXTRA_ALLOWED_CANONICALIZATIONS: ["http://www.w3.org/TR/2001/REC-xml-c14n-20010315#WithComments"]
+EXTRA_ALLOWED_CANONICALIZATIONS: http://www.w3.org/TR/2001/REC-xml-c14n-20010315#,http://www.w3.org/TR/2001/REC-xml-c14n-20010315#WithComments
```

Comment:
Turns out `env.list` doesn't parse actual stringified lists so I've updated this to correctly reflect the required input.